src/aspire/utils/misc.py

```diff
@@ -160,7 +161,7 @@ def gaussian_2d(size, mu=(0, 0), sigma=(1, 1), dtype=np.float64):
     return np.exp(-p).astype(dtype, copy=False)
 
 
-def gaussian_3d(size, mu=(0, 0, 0), sigma=(1, 1, 1), dtype=np.float64):
+def gaussian_3d(size, mu=(0, 0, 0), sigma=(1, 1, 1), indexing="zyx", dtype=np.float64):
```

Comment:
I'm going to make an issue about doing similar indexing for 2d. We don't need it now.